src/FluentValidation.Tests/FluentValidation.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
Please can you remove this? This is already being handled as part of the 11.0 upgrade, and it's preferred that PRs focus on one specific change only.